api/app/signals/apps/signals/api_generics/permissions.py

```diff
@@ -60,3 +61,24 @@ def has_permission(self, request, view):
             return True
         else:
             return False
+
+
+class SIAPermissions(BasePermission):
+    perms_map = {
+        'GET': ['signals.sia_read'],
```

Comment:
We should consider keeping only `sia_write` as `sia_read` is implied by the fact that the user is logged in.